src/qibocal/protocols/characterization/randomized_benchmarking/utils.py

```diff
@@ -12,16 +11,16 @@
     # Virtual gates
     0: gates.I,
     1: lambda q: gates.U3(q, 0, np.pi / 2, np.pi / 2),  # Z,
-    2: lambda q: gates.RZ(q, np.pi / 2),
-    3: lambda q: gates.RZ(q, -np.pi / 2),
+    2: lambda q: gates.U3(q, np.pi / 2, 0, 0),  # gates.RZ(q, np.pi / 2),
+    3: lambda q: gates.U3(q, -np.pi / 2, 0, 0),  # gates.RZ(q, -np.pi / 2),
```

Comment:
I think that this should be the correct decomposition according to the qibo documentation.
```suggestion
    2: lambda q: gates.U3(q, 0, np.pi / 2, 0),  # gates.RZ(q, np.pi / 2),
    3: lambda q: gates.U3(q, 0, -np.pi / 2, 0),  # gates.RZ(q, -np.pi / 2),
```
However, this doesn't fix the problem related to the RB above.